vcx/dummy-cloud-agent/src/actors/router.rs

```diff
@@ -25,11 +31,12 @@ use crate::utils::futures::*;
 pub struct Router {
     routes: HashMap<String, Recipient<HandleA2AMsg>>,
     pairwise_routes: HashMap<String, Recipient<HandleA2ConnMsg>>,
-    requester: Addr<Requester>
+    requester: Addr<Requester>,
+    fw_agent_wallet_handle: WalletHandle
```

Comment:
why is wallet_handle need to the router?

Reply:
Thanks, indeed not needed, forgot to remove this. Pushing commit to remove this now.